aioredis/connection.py

```diff
@@ -99,15 +104,20 @@ def create_connection(address, *, db=None, password=None, ssl=None,
 class RedisConnection:
     """Redis connection."""
 
-    def __init__(self, reader, writer, *, address, encoding=None, loop=None):
+    def __init__(self, reader, writer, *, address, encoding=None,
+                 parser=None, loop=None):
         if loop is None:
             loop = asyncio.get_event_loop()
+        if parser is None:
+            parser = Reader
+        assert callable(parser), (
```

Comment:
well, it's more like a fool-proof. I think we can add proper check later, if needed...